code/datums/components/tamed_command.dm

```diff
@@ -13,8 +13,8 @@
 		//Add riding component
 		M.tame = TRUE
 		M.can_buckle = TRUE
-		M.buckle_lying = FALSE
-		var/datum/component/riding/D = M.LoadComponent(/datum/component/riding/tamed)
+		M.buckle_lying = 0
```

Comment:
This should be false and not 0, or it should be a define that describes it

Reply:
> This should be false and not 0, or it should be a define that describes it

Its not a boolean, nor would it be proper as a define. 

Its very specifically degrees. 90, 270, 360, 180. That sort of thing. It would be redundant to add defines for that.

Someone made it 'false' because they thought it was a boolean and didnt know what they were doing.